### ISSUE TITLE
How to tag Lambda functions created from CloudCustodian

### ISSUE THREAD

**Issue by techsami** [cloud-custodian/cloud-custodian#3966]
Please suggest how to tag Lambda functions created by CloudCustodian

**Comment by thisisshi** [comment]
```yaml
mode:
  type: cloudtrail
  events:
    - CreateBucket
  tags:
    Application: Custodian
    CreatedBy: CloudCustodian
```

**Comment by kapilt** [comment]
we should add this to docs

**Comment by bclements** [comment]
Should we make a faq/kB section that is searchable?

**Comment by thisisshi** [comment]
at the very least in here: https://cloudcustodian.io/docs/policy/lambda.html

**Comment by thisisshi** [comment]
(also needs a mugc shoutout)

**Comment by tuxinaut** [comment]
> we should add this to docs

This would be awesome, spend quite some time to make it work.

**Comment by kapilt** [comment]
@bclements docs are already searchable

**Comment by kapilt** [comment]
else getting folks to use the custodian reddit for reference able questions faqs.. reddit.com/r/cloudcustodian

**Comment by kapilt** [comment]
closed in #4000
